common/frame_session.go

```diff
@@ -515,9 +515,11 @@ func (fs *FrameSession) onConsoleAPICalled(event *cdpruntime.EventConsoleAPICall
 		WithTime(event.Timestamp.Time()).
 		WithField("source", "browser-console-api")
 
-	if s := fs.vu.State(); s.Group.Path != "" {
-		l = l.WithField("group", s.Group.Path)
-	}
+		/* accessing the state Group while not on the eventloop is racy
+		if s := fs.vu.State(); s.Group.Path != "" {
+			l = l.WithField("group", s.Group.Path)
+		}
+		*/
```

Comment:
Should we remove the comment?

```suggestion
```

Reply:
I'd prefer to have them left commented out and maybe even link to a issue so that we know why it has been commented out and what it will take to uncomment them.

Reply:
> if we can make this code ot run on the event loop - that will make it possible to keep the lines, which is why I just commented them

👍